permissions/resource_permissions.go

```diff
@@ -133,6 +133,44 @@ func (a PermissionsAPI) shouldExplicitlyGrantCallingUserManagePermissions(object
 	return isDbsqlPermissionsWorkaroundNecessary(objectID)
 }
 
+func isOwnershipWorkaroundNecessary(objectID string) bool {
+	return strings.HasPrefix(objectID, "/jobs") || strings.HasPrefix(objectID, "/pipelines") || strings.HasPrefix(objectID, "/sql/warehouses")
+}
+
+func (a PermissionsAPI) getObjectCreator(objectID string) (string, error) {
+	w, err := a.client.WorkspaceClient()
+	if err != nil {
+		return "", err
+	}
+	if strings.HasPrefix(objectID, "/jobs") {
+		jobId, err := strconv.ParseInt(strings.ReplaceAll(objectID, "/jobs/", ""), 10, 64)
+		if err != nil {
+			return "", err
+		}
+		job, err := w.Jobs.GetByJobId(a.context, jobId)
+		if err != nil {
+			if strings.HasSuffix(err.Error(), " does not exist.") {
+				return "", nil
+			}
+			return "", err
+		}
+		return job.CreatorUserName, nil
+	} else if strings.HasPrefix(objectID, "/pipelines") {
+		pipeline, err := w.Pipelines.GetByPipelineId(a.context, strings.ReplaceAll(objectID, "/pipelines/", ""))
+		if err != nil {
+			return "", err
+		}
```

Comment:
Should we handle the "doesn't exist" case for the rest of objects?

Reply:
added the error suppression